python/pyspark/worker.py

```diff
@@ -160,6 +160,10 @@ def wrap_arrow_batch_udf(f, args_offsets, kwargs_offsets, return_type, runner_co
     import pandas as pd
 
     func, args_kwargs_offsets = wrap_kwargs_support(f, args_offsets, kwargs_offsets)
+    zero_arg_exec = False
+    if len(args_kwargs_offsets) == 0:
+        args_kwargs_offsets = (0,)  # Series([pyspark._NoValue, ...]) is used for 0-arg execution.
```

Comment:
This comment may not be always true, e.g., when there are 0-arg and 1+arg UDFs are mixed. I guess it still works, though.

Reply:
We need to add such tests if not exists.
- 0-arg only
- 0-arg, 1+arg
- 1+arg, 0-arg